pandas/core/sparse/frame.py

```diff
@@ -28,6 +28,13 @@
 from pandas.core.sparse.series import SparseSeries
 
 _shared_doc_kwargs = dict(klass='SparseDataFrame')
+depr_msg = """\
+SparseDataFrame is deprecated and will be removed in a future version.
+Use a DataFrame with sparse values instead.
```

Comment:
```suggestion
Use a regular DataFrame with sparse column values instead.
```

Not sure this is an improvement, though. I was only thinking that for us "DataFrame with sparse values" is very clear nowadays, but for somebody with less familiarity to the recent changes, it might be less clear (so that's at least my purpose of the proposed change)

Reply:
How about "regular DataFrame whose columns are SparseArrays"?